### DISCUSSION TITLE
alist播放迅雷怎么没有速度？

### DISCUSSION THREAD

**Discussion by sunzhe1234** [AlistGo/alist#1436] [Q&A]
我用fileball添加迅雷,都有速度

**Comment by sunzhe1234** [comment]
![99B02D58-5F0C-43A0-83A0-B89F1B384754](https://user-images.githubusercontent.com/100278321/182119883-22d4a9cc-7d41-4c2d-be6c-c112bba8d4a6.png)
![C8E40870-A2BE-4CD3-98F0-8E2FE99AB67F](https://user-images.githubusercontent.com/100278321/182119908-1c081f1e-d089-4b59-829e-c7f09cf21132.png)

**Comment by xhofe** [comment]
不再支持 将在下个版本移除。

**Reply by rakaki** [reply]
我用迅雷还是有速度的，种子离线的番剧都能正常看，希望作者大大不要移除。离线下载方面迅雷比百度和夸克都要强得多
